nncf/common/tensor_statistics/collectors.py

```diff
@@ -49,6 +49,8 @@ def register_input(self, x: TensorType) -> TensorType:
         """Registers input tensor"""
         if not self._enabled or \
                 self._num_samples is not None and self._collected_samples >= self._num_samples:
+            logger.debug(
+                'Trying to register input, but the number of samples is exceeded. So the tensor will not be registered')
```

Comment:
The message confuses me a little because of the 'not self._enabled' condition. Could you reformulate it something like that 'The input tensor is skipped' or remove.

Reply:
Fixed